nixos/modules/services/networking/netbird.md

```diff
@@ -2,55 +2,78 @@
 
 ## Quickstart {#module-services-netbird-quickstart}
 
-The absolute minimal configuration for the netbird daemon looks like this:
+The absolute minimal configuration for the Netbird client daemon looks like this:
 
 ```nix
 services.netbird.enable = true;
 ```
 
-This will set up a netbird service listening on the port `51820` associated to the
-`wt0` interface.
-
-It is strictly equivalent to setting:
+Which is equivalent to:
 
 ```nix
-services.netbird.tunnels.wt0.stateDir = "netbird";
+services.netbird.clients.wt0 = {
+  port = 51820;
+  name = "netbird";
+  interface = "wt0";
+  hardened = false;
+};
 ```
 
-The `enable` option is mainly kept for backward compatibility, as defining netbird
-tunnels through the `tunnels` option is more expressive.
+This will set up a `netbird.service` listening on the port `51820` associated to the
+`wt0` interface. There will also be `netbird-wt0` binary installed in addition to `netbird`.
```

Comment:
Using the config from https://docs.netbird.io/how-to/installation#linux
```nix
 { config, pkgs, ... }:
 {
   services.netbird.enable = true; # for netbird service & CLI
   environment.systemPackages = [ pkgs.netbird-ui ]; # for GUI
 }
```

And then running as described in https://docs.netbird.io/how-to/installation#running-net-bird-with-sso-login it doesn't work as expected:
```
% netbird up
netbird: command not found
```

Reply:
created an alias for `netbird` and `netbird-ui` command